models/models.js

```diff
@@ -24,4 +22,21 @@ exports.getUsersM = () => {
   return db.query(`SELECT * FROM users;`).then(({ rows: users }) => {
     return users;
   });
-}
\ No newline at end of file
+};
+
+exports.patchReviewVoteM = (id, votes) => {
+  return db
+    .query(
+      `UPDATE reviews
+SET votes = votes + $1
+WHERE review_id = $2
+RETURNING  *;`,
+      [votes, id]
+    )
+    .then(({ rows: review }) => {
```

Comment:
It doesn't quite make sense to recast rows as review but you could use nested destructuring here instead eg `{rows: [review]}`... then review will have the value of rows[0]